### PR TITLE
chore: clean up init and upgrade express server

### PR DESCRIPTION
**Pull request by cesara**

### CODE REVIEW

File: ts-web-component/src/index.ts [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+import {mutators} from '../../shared/mutators';
 import {MyTodo} from './component/my-todo';
 import {TodoItem} from './component/todo-item';
+import {createSpace, spaceExists} from './space';
 import './style.css';
 
 window.customElements.define('my-todo', MyTodo);
 window.customElements.define('todo-item', TodoItem);
+
+async function init() {
```
Comment (cesara) [inline ts-web-component/src/index.ts:11]: replicache init is non DRY at the moment. I need to figure out the correct directory structure to handle this well. The current shared folder is for server-client shared files.

File: react/src/index.tsx [hunk 2]
```diff
@@ -4,81 +4,57 @@ import './index.css';
...
+    window.location.href = '/list/' + (await createSpace());
+    return;
   }
-};
 
-async function spaceExists(spaceID: string): Promise<boolean> {
-  const spaceExistRes = await fetchJSON('spaceExists', spaceID);
+  // URL layout is "/list/<listid>"
+  const paths = pathname.split('/');
+    const [, listDir, listID] = paths;
```
Comment (aboodman) [inline react/src/index.tsx:19]: indent?

**Review by aboodman — APPROVED** [review]
Think you might need to run format though.

File: react/src/index.tsx (hunk 2 above)
Comment (cesara) [reply react/src/index.tsx]: fixed, thanks

**Pull request merged**